From 889fdc92bf49ce55cd5adfd753b4047081803c4d Mon Sep 17 00:00:00 2001
From: Vincent Davis Jr <vince@underview.tech>
Date: Mon, 30 May 2022 10:45:09 -0500
Subject: [PATCH] CMakeLists: comment add_subdirectory(third_party) line

Upstream-Status: Inappropriate

Not required as we changes where to look for headers
for packages:
	* spirv-headers
	* spirv-tools
	* glslang

Signed-off-by: Vincent Davis Jr <vince@underview.tech>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b55e1f0..262af69 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -124,7 +124,7 @@ endif(MSVC)
 
 # Configure subdirectories.
 # We depend on these for later projects, so they should come first.
-add_subdirectory(third_party)
+#add_subdirectory(third_party)
 
 add_subdirectory(libshaderc_util)
 add_subdirectory(libshaderc)
-- 
2.25.1

